.github/CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # Each line is a file pattern followed by one or more owners.
 
 # global
-* @buchi-busireddy @tim-mwangi @inespo @avinashkolluru @kotharironak
+* @hypertrace/config-services-owners
```

Comment:
Updated since I was in here

Reply:
Nevermind, reverted to unblock this change since it's a holiday in India and I'm the only code owner not based in India :) Pulled into a separate PR: https://github.com/hypertrace/entity-service/pull/101